README.md

```diff
@@ -1 +1,9 @@
-# document-database
\ No newline at end of file
+# document-database
+
+This needs to have:
```

Comment:
Please use the example I provided for guidance on what you should do in this file. This is ground zero, the first step before you code.